### PR TITLE
weekday condition

### PR DESCRIPTION
**Pull request by Misiu**
## Proposed change

I've documented splitting time condition into two: time (with `before` and `after` properties) and weekday (with `days` property)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/32848, https://github.com/home-assistant/frontend/pull/5324
- This PR fixes or closes issue:

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_docs/scripts/conditions.markdown [hunk 1]
```diff
@@ -239,10 +240,31 @@ Note that if only `before` key is used, the condition will be `true` *from midni
 If only `after` key is used, the condition will be `true` from the specified time *until midnight*.
 Time condition windows can span across the midnight threshold if **both** `after` and `before` keys are used. In the example above, the condition window is from 3pm to 2am.
 
-<div class='note tip'>
+<div class='note warning'>
+As of release 0.109, the weekday property will be deprecated. Replace it with weekday condition.
```
Comment (frenck) [inline source/_docs/scripts/conditions.markdown]: We don't keep change log warnings/notes in the documentation. Those go into the breaking changes section of the release notes. Please remove it.

File: source/_docs/scripts/conditions.markdown [hunk 2]
```diff
@@ -239,10 +240,31 @@ Note that if only `before` key is used, the condition will be `true` *from midni
...
+</div>
 
+<div class='note tip'>
```
Comment (frenck) [inline source/_docs/scripts/conditions.markdown]: We don't have a "note tip". Is is really needed to make a large notification box here?

File: source/_docs/scripts/conditions.markdown [hunk 3]
```diff
@@ -228,6 +228,7 @@ condition:
   # At least one of the following is required.
   after: '15:00:00'
   before: '02:00:00'
+  # weekday property will be deprecated in release 0.109
```
Comment (frenck) [inline source/_docs/scripts/conditions.markdown]: Don't leave deprecated stuff in here...

File: source/_docs/scripts/conditions.markdown [hunk 4]
```diff
@@ -239,10 +240,31 @@ Note that if only `before` key is used, the condition will be `true` *from midni
...
+  condition: weekday
+  days:
+    - mon
+    - wed
+    - fri
+```
+
+Valid values for `days` are `mon`, `tue`, `wed`, `thu`, `fri`, `sat`, `sun`.
+
+<div class='note tip'>
```
Comment (frenck) [inline source/_docs/scripts/conditions.markdown]: Same comment as above... do we need to mention this twice btw?

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @Misiu, I've reviewed it and left some comments. Could you please take a look? Thanks! 👍

**Comment by frenck** [comment]
One thing I noticed, is that you committed this onto the `next` branch in your fork. Please note that it is strongly advised to create a feature branch on your fork and use that feature branch to open a PR with. It can safe you from quite a bit of trouble.

**Comment by Misiu** [comment]
@frenck I'll close this PR and create a new branch, as you suggested.
I wasn't sure if I should leave the deprecated config options.
In my PR (https://github.com/home-assistant/core/pull/32848) I didn't remove old options to keep them working for a while.
If you agree I can remove deprecated options and make this a breaking change.
What do you think?

**Comment by frenck** [comment]
Since you are committing to the `next` branch (which is the correct branch to use for this PR). Changes will be published when the parent PR (code) is published in a release.

File: source/_docs/scripts/conditions.markdown (hunk 3 above)
Comment (Misiu) [reply source/_docs/scripts/conditions.markdown]: ok, removed

File: source/_docs/scripts/conditions.markdown (hunk 1 above)
Comment (Misiu) [reply source/_docs/scripts/conditions.markdown]: removed

File: source/_docs/scripts/conditions.markdown (hunk 4 above)
Comment (Misiu) [reply source/_docs/scripts/conditions.markdown]: fixed

File: source/_docs/scripts/conditions.markdown [hunk 5]
```diff
@@ -220,29 +220,37 @@ Within an automation, template conditions also have access to the `trigger` vari
...
```
Comment (frenck) [inline source/_docs/scripts/conditions.markdown]: We don't have a tip message box.
Let's remove the whole message box, leaving just the contents as normal.

File: source/_docs/scripts/conditions.markdown (hunk 2 above)
Comment (Misiu) [reply source/_docs/scripts/conditions.markdown]: Currently, the time condition has this info: https://www.home-assistant.io/docs/scripts/conditions/#time-condition
I wanted to leave it "as is.

File: source/_docs/scripts/conditions.markdown (hunk 5 above)
Comment (Misiu) [reply source/_docs/scripts/conditions.markdown]: image
this is from docs page: https://www.home-assistant.io/docs/scripts/conditions/#time-condition

Comment (frenck) [reply source/_docs/scripts/conditions.markdown]: Sure, but `tip` still is invalid and not defined.

Comment (Misiu) [reply source/_docs/scripts/conditions.markdown]: I've removed the `tip` class. As I wrote I looked at the current version.
I can remove `tip` class in other places (across `_docs` folder) in a separate PR if needed.

File: source/_docs/scripts/conditions.markdown (hunk 2 above)
Comment (Misiu) [reply source/_docs/scripts/conditions.markdown]: I've removed the `tip` class

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by balloob** [comment]
Backend PR closed.

**Pull request closed without merging**